theano/sandbox/rng_mrg.py

```diff
@@ -1292,8 +1292,17 @@ def uniform(self, size, low=0.0, high=1.0, ndim=None, dtype=None,
             The output data type. If dtype is not specified, it will be
             inferred from the dtype of low and high, but will be at
             least as precise as floatX.
-
+        nstreams:
+            The number of streams to use for sampling the random numbers
+            in parallel.
+        broadcastable:
```

Comment:
I'm always kind of hesitant about being too "clever" about trying to infer what the user meant from incomplete information. Adding broadcastable flags based on simply the shape could introduce unexpected broadcast-related bugs. I'm much more comfortable with a solution that forces the user to specify broadcastability if that is what they want.